docs/pages/setup/reference/moderated-sessions.mdx

```diff
@@ -0,0 +1,115 @@
+---
+title: Moderated Sessions
+description: Moderated Sessions RBAC Reference
+h1: Moderated Sessions
+---
+
+# Moderated Sessions RBAC
+
+Moderated Sessions makes use of RBAC policies to allow for fine grained control over
+who can join a session and who is required to be present to start one.
+
+The system is based around **require policies** and **allow policies**. 
+
+Require policies define a set of conditions
+that must be a met for a session to start/run.
+
+Allow policies are used to define what sessions a user can join
+and under what conditions they may join a session.
+
+## Configuring moderated sessions
```

Comment:
My biggest issue with this section is that it is more like a guide than a reference. When reading a reference, I would expect a comprehensive set of available options (e.g., see our docs guide: https://goteleport.com/docs/docs/#reference-manuals). Instead, each section describes or illustrates a topic without being exhaustive.

I recommend reworking this section to have the following structure:
1. Add headers for `require_session_join` and `join_sessions`
1. Under each header, list the required/optional keys and their possible values (or the types of these values), as well as what these options configure. I suggest using a table.
1. Add a separate section explaining the filter syntax. The section should have the same heading level as the `require_session_join` and `join_sessions` sections. When we indicate the type of the `filter` option, we can link to that section.
1. Change "Session kinds" to use a table. We can link to this section when specifying the type of the `kinds` value.
1. Move the "Participant modes" section into the same heading level as the `join_sessions`/`require_session_join` sections. That way, when we indicate the type of the `modes` field, we can say that it's a list of participant modes, and link to that section.

As an illustration of what I mean, I think some parts of the Istio reference documentation do a good job at laying out tables of YAML config options and specifying the types of the values. In [this example](https://istio.io/latest/docs/reference/config/networking/gateway/#Gateway), we can see that the the `servers` field is a `Server[]`, and a separate section specifies the fields of `Server`.